platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -1310,11 +1310,17 @@ void DisplayServerX11::show_window(WindowID p_id) {
 	const WindowData &wd = windows[p_id];
 	popup_open(p_id);
 
+	const Point2i bmpos = wd.position;
+
 	DEBUG_LOG_X11("show_window: %lu (%u) \n", wd.x11_window, p_id);
 
 	XMapWindow(x11_display, wd.x11_window);
 	XSync(x11_display, False);
 	_validate_mode_on_map(p_id);
+	if (!wd.borderless && !wd.fullscreen && !wd.exclusive_fullscreen) {
+		usleep(50000); // waiting for the window-manager to compelete the decoration of the window
```

Comment:
There has to be a better solution, I got the idea (and it should work in most cases), but relying on delay is not a robust thing.

Reply:
Well, we can track the window's parent through notify configure sent by the window manager while decorating. I don't know if that will work.